probe/tag/tagger.go

```diff
@@ -0,0 +1,17 @@
+package tag
+
+import "github.com/weaveworks/scope/report"
+
+// Tagger tags nodes with value-add node metadata.
+type Tagger interface {
+	Tag(r report.Report) report.Report
+	Stop()
```

Comment:
Given not ever Tagger implements Stop, I suggest we remove it from the interface, return a concrete DockerMapper, and then defer docker.Stop().